WordPress/src/main/java/org/wordpress/android/ui/JetpackUtils.java

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.ui;
+
+import org.wordpress.android.analytics.AnalyticsTracker;
+
+import java.util.HashMap;
+
+/**
+ * Wraps utility methods for Jetpack
+ */
+public class JetpackUtils {
+    private JetpackUtils() {
```

Comment:
I don't think this constructor is needed.

Reply:
Nope but it's usually a standard for a utility class to have a private constructor (and only static methods). It stops people from initialising it. I could remove it if you don't like it.

Reply:
Same idea with [this](https://github.com/wordpress-mobile/WordPress-Android/pull/7331#discussion_r173816863), I like consistency and even if a developer accidentally instantiates it, they won't actually be able to do anything with it. But again, if you think this is better to do, we can keep it as is.